task.go

```diff
@@ -119,16 +119,17 @@ func (e *Executor) Setup() error {
 			Vars:   e.taskvars,
 			Logger: e.Logger,
 		}
-	case version.IsV2(v), version.IsV21(v), version.IsV22(v):
+	case version.IsV2(v), version.IsV21(v), version.IsV22(v), version.IsV23(v):
```

Comment:
@andreynering Should this be changed to `version.isV26(v)`?   I'm a little confused about how this version detection works - since if I change the point version in my actual Taskfiles I get "X is only available on <older version>"

Reply:
minimal repro:

```
---
version: '2.3'
output: prefixed
```

```
task: Taskfile option "output" is only available starting on Taskfile version v2.1
```

This _seems_ like a bug to me in that '2.3' is obviously > 2.1 - but maybe the version checking is only against the major version?

Reply:
@stephenprater Indeed, the version code needs some love.

In theory this should actually be:

```go
case version.IsV2(v), version.IsV21(v), version.IsV22(v), version.IsV23(v), version.IsV24(v), version.IsV25(v), version.IsV26(v):
```

But there's likely a way to create a specific constraint for `>= 2, < 3`.

I'd say, let this as is, and I'll add a reminder to fix this once I merge this PR to master. 🙂